spec/Dockerfile.puppetdb

```diff
@@ -0,0 +1,22 @@
+FROM puppet/puppetdb
+
+ARG hostname="bolt-puppetdb"
+
+#ENV PUPPETSERVER_HOSTNAME "$hostname"
+
```

Comment:
```suggestion
```

Reply:
Just an extra newline

Reply:
It's set so that we can get a certificate from puppetserver. The whole reason we have our own dockerfile is because we don't want to get a certificate from puppetserver. I'll delete the line.